src/main/java/com/es/phoneshop/controller/servlet/ProductDetailsPageServlet.java

```diff
@@ -0,0 +1,32 @@
+package com.es.phoneshop.controller.servlet;
+
+import com.es.phoneshop.model.dao.impl.ArrayListProductDao;
+import com.es.phoneshop.model.exception.ItemNotFoundException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ProductDetailsPageServlet extends HttpServlet {
+    private static final String PRODUCT_CODE_ATTRIBUTE = "product";
+    private static final String ERROR_MESSAGE_ATTRIBUTE = "message";
+    private static final String PRODUCT_DETAILS_PATH = "/WEB-INF/pages/productDetails.jsp";
+    private static final String ERROR_MESSAGE_PART_1 = "Product with code '";
```

Comment:
Probably it would be better to go with a single "something %s something" format string and make use of String.format() in doGet method